diff --git a/cocoapods-patch-20220817-22377-15nosdt/M5MultitouchSupport/M5MultitouchSupport/M5MultitouchManager.m b/Pods/M5MultitouchSupport/M5MultitouchSupport/M5MultitouchManager.m
index c6fda74..a5bfdb9 100644
--- a/cocoapods-patch-20220817-22377-15nosdt/M5MultitouchSupport/M5MultitouchSupport/M5MultitouchManager.m
+++ b/Pods/M5MultitouchSupport/M5MultitouchSupport/M5MultitouchManager.m
@@ -231,7 +231,7 @@ static BOOL laptopLidClosed = NO;
             MTDeviceRef mtDevice = (__bridge MTDeviceRef)device;
             MTUnregisterContactFrameCallback(mtDevice, mtEventHandler);
             MTDeviceStop(mtDevice);
-            MTDeviceRelease(mtDevice);
+//            MTDeviceRelease(mtDevice);
         } @catch (NSException *exception) {}
     }
 }
